c_src/device/bcm.c

```diff
@@ -251,33 +253,33 @@ void device_poll()
 
 void device_begin_render(driver_data_t* p_data)
 {
-  NVGcontext* p_ctx = p_data->p_ctx;
+  NVGcontext* v_ctx = p_data->v_ctx;
 
   glClear(GL_COLOR_BUFFER_BIT);
 
-  nvgBeginFrame(p_ctx, g_device_info.width, g_device_info.height, g_device_info.ratio);
+  nvgBeginFrame(v_ctx, g_device_info.width, g_device_info.height, g_device_info.ratio);
 
   // set the global transform
-  nvgTransform(p_ctx,
+  nvgTransform(v_ctx,
                p_data->global_tx[0], p_data->global_tx[1],
                p_data->global_tx[2], p_data->global_tx[3],
                p_data->global_tx[4], p_data->global_tx[5]);
 }
 
 void device_begin_cursor_render(driver_data_t* p_data)
 {
-  NVGcontext* p_ctx = p_data->p_ctx;
-	nvgTranslate(p_ctx,
+  NVGcontext* v_ctx = p_data->v_ctx;
```

Comment:
```suggestion
  NVGcontext* p_ctx = (NVGcontext*)p_data->v_ctx;
```
then the rest of function doesn't need to change

Same pattern for other functions

Reply:
Makes sense (re https://github.com/ScenicFramework/scenic_driver_local/issues/48#issuecomment-1705606724). Updated in 37e1b7d982074af512ae06036103094a56157b6f